core/trino-spi/src/main/java/io/trino/spi/block/ArrayBlock.java

```diff
@@ -369,11 +369,15 @@ public <T> T getObject(int position, Class<T> clazz)
         if (clazz != Block.class) {
             throw new IllegalArgumentException("clazz must be Block.class");
         }
-        checkReadablePosition(this, position);
+        return clazz.cast(getArray(position));
+    }
 
+    public Block getArray(int position)
```

Comment:
Why don't all of these method additions need revapi changes? Maybe it doesn't flag on new methods?

Reply:
We only check for backwards incompatible changes